VotingApplication/VotingApplication.Web/Views/Home/Index.cshtml

```diff
@@ -7,8 +7,8 @@
 <body>
     <div class="center-panel">
         <div class="body-content">
-            <h2>Where do we go for lunch?</h2>
-            <p id="expiry-date">Poll closes in: 5 hours</p>
+            <h2>{{pollName}}</h2>
```

Comment:
Small point, bit if I recall correctly, we will encourage the poll name to be a question by putting a question mark after the name of the poll during poll creation, so perhaps one should be put here?
i.e.
{{pollName}}?